README.md

```diff
@@ -86,39 +86,7 @@ The `import.js` file you provide will be bundled with any locally referenced scr
 
 ### Assistant
 
-Run AI-enabled commands to assist with your import script development.
-
-Add an npm script entry to your Edge Delivery project's `package.json`:
-
-```
-"assistant": "aem-import-helper assistant"
-```
-
-Run the script:
-
-```
-npm run assistant -- start --url https://example.com --outputPath tools/importer
-npm run assistant -- cleanup --url https://example.com --prompt "content to remove"
-npm run assistant -- block --url https://example.com --name "name of the block" --prompt "describe block content on the page"
-npm run assistant -- cells --url https://example.com --name "name of the block" --prompt "describe content that should be added to the block"
-```
-
-#### Commands
-
-- `start`: Start a new import project
-- `cleanup`: Remove content from the page
-- `block`: Define a block on the page
-- `cells`: Add content to a block
-
-#### Options
-
-- `--url`: The URL of the page to analyze
-- `--outputPath`: The directory to save the import scripts
-- `--name`: The name of the block
-- `--prompt`: Descriptive text to help understand the content
-
-#### Authentication
-
+See [Import Assistant](./docs/import-assistant.md) for usage.
 
 
 ## Coming soon
```

Comment:
Is this still coming soon?  Might already be here:  @blefebvre @swetabar 
(Don't hold up the merge for it, but...)

Reply:
Already implemented — that line (and the entire section for now) can be removed 👍  Nice catch.